templates/user/notification/notification_subscriptions.tmpl

```diff
@@ -0,0 +1,79 @@
+{{template "base/head" .}}
+<div class="page-content user notification" id="notification_subscriptions" data-params="{{.Page.GetParams}}" data-sequence-number="{{.SequenceNumber}}">
+	<div class="ui container">
+		<div class="ui top attached tabular menu">
+			<a href="{{AppSubUrl}}/notifications/subscriptions" class="{{if eq .Status 1}}active{{end}} item">
```

Comment:
Please change all these to `{{if eq .Status 1}}active {{end}}item`, that way we don't end up with one usesless space character :)